packages/sdk/src/resolve/resolve-accounts.js

```diff
@@ -4,8 +4,53 @@ import {log} from "@onflow/util-logger"
 import {isTransaction} from "../interaction/interaction.js"
 import {createSignableVoucher} from "./voucher.js"
 
+const MAX_DEPTH_LIMIT = 5
+
+const CHARS = "abcdefghijklmnopqrstuvwxyz0123456789".split("")
+const randChar = () => CHARS[~~(Math.random() * CHARS.length)]
+const uuid = () => Array.from({length: 10}, randChar).join("")
```

Comment:
event thought is uuid is used as a backup if no tempId. Is there a chance of uuid collisions since we are creating our own?